assets/js/blocks/attribute-filter/index.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { registerBlockType } from '@wordpress/blocks';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit.js';
+
+registerBlockType( 'woocommerce/attribute-filter', {
+	title: __( 'Filter Products by Attribute', 'woo-gutenberg-products-block' ),
+	icon: {
+		src: <Gridicon icon="menus" />,
+		foreground: '#96588a',
+	},
+	category: 'woocommerce',
+	keywords: [ __( 'WooCommerce', 'woo-gutenberg-products-block' ) ],
+	description: __(
+		'Display a list of filters based on a chosen product attribute.',
+		'woo-gutenberg-products-block'
+	),
+	supports: {
+		align: [ 'wide', 'full' ],
+	},
+	attributes: {
+		attributeId: {
+			type: 'number',
+			default: 0,
+		},
+		showCounts: {
+			type: 'boolean',
+			default: true,
+		},
+		queryType: {
+			type: 'string',
+			default: 'and',
+		},
+	},
+	edit,
+	/**
+	 * Save the props to post content.
+	 */
+	save( { attributes } ) {
+		const { showCounts, displayStyle, queryType, attributeId } = attributes;
+		const data = {
+			'data-attributeid': attributeId,
```

Comment:
Is there any preference on using `data-attributeid` instead of `data-attribute-id`? Same for the attributes below.

Reply:
I'd prefer hyphenating all words.  That will result in clearer camelcase properties too.